.github/wiki/_Sidebar.md

```diff
@@ -1,6 +1,7 @@
 ## Table of Contents
 
 - [[Home]]
+- [[Installation]]
```

Comment:
the `Installation` term might confuse with the install the template itself (https://github.com/nimblehq/elixir-templates#installation) in a new project part, should we find another term? something like `generate a sample project`

How about? `Test the apply action on development`

Reply:
Fixed in ba44478